### DISCUSSION TITLE
Table Vertical headers

### DISCUSSION THREAD

**Discussion by MattDowningShell** [ant-design-blazor/ant-design-blazor#2491] [Q&A]
Hi,
I did search but couldn't see an answer.  I'm after tabulated data (Table) but flowing horizontally (headers to the left).
As is achieved in Html via:
```
<table>
<tr><th>my heading</th><td>data1</td><td>data2</td></tr>
```
etc
How can I do this with the Table component?
Thank you!

**Comment by ElderJames** [comment]
Please use the `Descriptions` component
